### ISSUE TITLE
Bug: Burgundy not losing its colonies

### ISSUE THREAD

**Issue by tbg2021** [TheDeNuke/DvD-Custom-Branch#15]
If the Dutch revolt but Burgundy holds onto Pas de Calais, they will apparently keep their colonies after being landlocked (at least if France happens to be the one to conquer Calais later).

**Comment by TheDeNuke** [comment]
The code for this script is very simple.

allow = { 
		owns = 388
		owns = 378
		owns = 375
		owns = 383
		OR = {
		owns = 399
		FRA = { owns = 399 } 
		DUA = { owns = 399 }
		CHA = { owns = 399 }
		}
		OR = {
		owns = 402
		FRA = { owns = 402 } 
		DUA = { owns = 402 }
		CHA = { owns = 402 }
		}
		BRG = { NOT = { owns = 467 } }
		OR = { 
		AND = { 
			BRG = { is_greater_power = no }
			is_secondary_power = yes 
				}
		AND = { 
			BRG = { is_greater_power = yes }
			is_greater_power = yes 
				}
			}
		}

The issue that you are likely seeing is the second half of the code and not the code that requires specific provinces to be owned by the Netherlands (since it accounts for France owning the territory as well). Burgundy **must** implicitly decline as a empire before the Netherlands is able to take their territory.

Basically, its WAD. If you want a quick test then reduce the Prestige of Burgundy to Zero and watch what happens.
